interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java

```diff
@@ -172,29 +177,30 @@ private void tearDown() {
     }
   }
 
-  private static void runShellCmd(String cmd) throws Exception {
+  private static void runShellCmd(String cmd, boolean skipNetCmd) throws Exception {
```

Comment:
I would suggest make the method non-static, so you don't have to add a boolean argument.

Reply:
Done